### PR TITLE
Changed the order of csv content's columns to match the ones from the app

### PR DESCRIPTION
**Pull request by santostiago**

### CODE REVIEW

File: app/services/api/v1/data/emission_pathways/filter.rb [hunk 1]
```diff
@@ -34,7 +34,7 @@ def initialize(params)
           def call
             @query = apply_filters(@query, mview: true)
             @years_query = apply_filters(@years_query, mview: false)
-            @years = @years_query.distinct(:year).pluck(:year).sort
+            @years = @years_query.distinct(:year).pluck(:year).sort.reverse
```
Comment (agnessa) [inline app/services/api/v1/data/emission_pathways/filter.rb]: sure? certainly the opposite was requested two months ago 😕  https://www.pivotaltracker.com/story/show/159961025

File: app/services/api/v1/data/emission_pathways/filter_columns.rb [hunk 2]
```diff
@@ -48,10 +44,6 @@ def select_columns_map
                 column: 'composite_name',
                 alias: 'composite_name'
               },
-              {
-                column: 'unit',
```
Comment (agnessa) [inline app/services/api/v1/data/emission_pathways/filter_columns.rb]: do we need to remove unit? I think it is displayed on the frontend?

File: app/services/api/v1/data/emission_pathways/filter.rb (hunk 1 above)
Comment (simaob) [reply app/services/api/v1/data/emission_pathways/filter.rb]: :(

File: app/services/api/v1/data/emission_pathways/filter_columns.rb (hunk 2 above)
Comment (simaob) [reply app/services/api/v1/data/emission_pathways/filter_columns.rb]: it was duplicated. there's another call to it further down.

**Pull request merged**